pkg/dataMover/data_mover.go

```diff
@@ -65,9 +68,9 @@ func (this *DataMover) CopyToRepo(peID astrolabe.ProtectedEntityID) (astrolabe.P
 		return astrolabe.ProtectedEntityID{}, err
 	}
 
-	this.Infof("Ready to call s3 PETM copy API")
+	this.Debugf("Ready to call s3 PETM copy API")
```

Comment:
Please add the PEID we're copying from

Reply:
Use log := this.WithField("Local PEID", peID.String()) with will contains the peID in each log message.